pytest_invenio/fixtures.py

```diff
@@ -69,6 +69,15 @@ def instance_path():
     shutil.rmtree(path)
 
 
+@pytest.fixture(scope="module")
+def cache_uri():
+    """Cache uri."""
+    if "CACHE_REDIS_URL" in os.environ:
+        yield os.environ["CACHE_REDIS_URL"]
+    else:
+        yield None
```

Comment:
minor: shouldn't this return a default `"redis://localhost:6379/"`? I'm looking at the rest of the service-related fixtures, and they all return some default value

Reply:
i think it will never be used. pytest-invenio install docker-services-cli which defines the configuration variable `CACHE_REDIS_URL` if that variable is not there it should break somehow because then it should be fixed in `docker-services-cli` 

if CACHE_REDIS_URL is None i think the redis libraries fall back to "redis://localhost:6379/"